test/5.public.HtmlChecker.js

```diff
@@ -302,18 +302,36 @@ describe("PUBLIC -- HtmlChecker", function()
 				}
 			}).scan(htmlString, baseUrl);
 		});
-		
-		
-		
+
+		it("excludeExternalLinks = true", function(done)
+		{
+			var htmlString = '<a href="https://example.com">link1</a>';
+			htmlString += '<a href="https://example.com">link2</a>';
```

Comment:
One link should be sufficient, especially considering that both of these are identical.